CHANGELOG.md

```diff
@@ -13,6 +13,9 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
 - **UIView**
   - Added `removeBlur()` method for removing the applied blur effect from the view. [#1159](https://github.com/SwifterSwift/SwifterSwift/pull/1159) by [regi93](https://github.com/regi93)
   - Added `makeCircle(diameter:)` method to make the view circular. [#1165](https://github.com/SwifterSwift/SwifterSwift/pull/1165) by [happyduck-git](https://github.com/happyduck-git)
+  - **NSView**
```

Comment:
Indentation
```suggestion
- **NSView**
```

Reply:
Also, can you move these up to the "Upcoming Release"? It's currently empty but you should move it there.